docs/source/javascripts/all.js

```diff
@@ -4,21 +4,47 @@ places({
   container: document.querySelector('#landing-demo')
 });
 
+// The following code is used to change the color of the navigation
+// depending the level of page scroll.
+const hero = document.querySelector('.hero-section');
+const navigation = document.querySelector('.navigation') ;
+const height = hero.offsetHeight;
+const navHeight = navigation.offsetHeight;
+const sidebar = document.getElementById('sidebar');
+
+
 // automatically darken the top menu when going down
 document.addEventListener('scroll', scrollEvent => {
   const hero = document.querySelector('.hero-section');
   const navigation = document.querySelector('.navigation');
+	const value = event.target.scrollingElement.scrollTop;
```

Comment:
Bad indent, we use two spaces, you can configure your editor I guess to always use two spaces